config/bidderinfo.go

```diff
@@ -243,10 +252,49 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 				return nil, fmt.Errorf("error parsing config for bidder %s: %v", fileName, err)
 			}
 
-			infos[string(normalizedBidderName)] = info
+			bidderInfos[string(normalizedBidderName)] = info
+
+			//need to maintain nullable fields from BidderInfo struct into bidderInfoNullableFields
+			//to handle the default values in aliases yaml
+			if len(info.AliasOf) > 0 {
+				aliasInfo := bidderInfoNullableFields{}
+				if err := yaml.Unmarshal(data, &aliasInfo); err != nil {
+					return nil, fmt.Errorf("error parsing config for aliased bidder %s: %v", fileName, err)
+				}
+
+				aliasInfos[string(normalizedBidderName)] = aliasInfo
```

Comment:
What do we do with the `aliasInfo` struct we are saving here? You pass `aliasInfos` in to `processBidderAliases()`, but that only uses `aliasInfos` to loop over the keys of the map. I don't see where you are accessing the struct you have stored in the map.

Reply:
There is a PR dependent on this one for conditional overwrites: https://github.com/prebid/prebid-server/pull/3001